aws/inputseeders/aws/masterdata.json

```diff
@@ -946,6 +956,127 @@
                     }
                 }
             }
+        },
+        "_awswin" : {
+            "Modes" : {
+                "*" : {
+                    "ecs" : {
+                        "Profiles" : {
+                            "Storage" : "ecs_awswin"
+                        },
+                        "ComputeInstance" : {
+                            "OperatingSystem" : {
+                                "Family" : "windows",
+                                "Distribution" : "awswin",
+                                "MajorVersion" : "2019"
+                            },
+                            "Image" : {
+                                "Source" : "aws:SSMParam",
+                                "aws:Source:SSMParam" : {
+                                    "Name" : "/aws/service/ami-windows-latest/Windows_Server-2019-English-Full-ECS_Optimized"
+                                }
+                            },
+                            "OSPatching": {
+                                "Enabled":false
+                            },
+                            "ComputeTasks": [
+                                "_computetask_awscli",
+                                "_computetask_awswin_cfninit",
+                                "_computetask_awswin_userbootstrap",
+                                "_computetask_awswin_hamletenv",
+                                "_computetask_awswin_volumemount",
+                                "_computetask_awswin_filedir",
+                                "_computetask_awswin_vpc_lb",
+                                "_computetask_awswin_notavailable"
+                            ]
+                        }
+                    },
+                    "computecluster" : {
+                        "ComputeInstance" : {
+                            "OperatingSystem" : {
+                                "Family" : "windows",
+                                "Distribution" : "awswin",
```

Comment:
I don't think this is a real distribution. We should reserve this for when its required.

Reply:
Since it's not a vanilla windows install I was appropriating the term "distribution" to describe the AWS customised windows install (AWS CLI, EC2Lauch etc). It's not as if there is (or likely ever will be - would need MS to opensource Windows) a Redhat Windows and a MS Windows, unlike Linux.

The other potential "distribution" I thought about was the "preview" windows versions from AWS. I was thinking of awswin and awspwin, but they could be "preview" and "ga" distributions just as easily. Thoughts?